lib/rules/no-magic-numbers.js

```diff
@@ -90,6 +95,18 @@ module.exports = {
             return ignore.indexOf(value) !== -1;
         }
 
+        /**
+         * Returns whether the number should be ignored when used as a default
+         * value assignment.
```

Comment:
Can you please update this description, this function no longer checks whether the number should be ignored.

Reply:
Done.